project-saarah/scenes/select-product-measurement/SelectProductMeasurementRouter.swift

```diff
@@ -14,29 +14,30 @@ protocol SelectProductMeasurementRoutingLogic {
 
 protocol SelectProductMeasurementDataPassing {
 	var dataStore: SelectProductMeasurementDataStore? { get }
+    var selectMeasureDataStore: SelectedMeasureReceptor? { get set }
```

Comment:
The receptor protocol must be implemented by the data store, and not be property of router DataPassing.